x-pack/plugins/lens/public/react_embeddable/inline_editing/panel_management.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { apiIsPresentationContainer } from '@kbn/presentation-containers';
+import { BehaviorSubject } from 'rxjs';
+import { LensRuntimeState } from '../types';
+
+export interface PanelManagementApi {
+  isEditingEnabled: () => boolean;
+  isNewPanel: () => boolean;
+  onStopEditing: (isCancel: boolean, state: LensRuntimeState | undefined) => void;
+}
+
+export function setupPanelManagement(
```

Comment:
where is the name coming from? I am trying to understand what is panel management as a concept.

Reply:
The previous inline editing component had the concept of `panel` (see `isNewPanel` prop, etc...), I've kept that around for now.